src/runtime/c_runtime_api.cc

```diff
@@ -105,6 +105,9 @@ class DeviceAPIManager {
   DeviceAPIManager() {
     std::fill(api_.begin(), api_.end(), nullptr);
   }
+  ~DeviceAPIManager() {
```

Comment:
For this change and the one below, can we just send a PR to upstream?

Reply:
Sure I can open a PR there too

Reply:
Done: https://github.com/apache/incubator-tvm/pull/5986